src/Containers/SearchContainer.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import SearchBar from "../Components/SearchBar";
+import axios from 'axios';
+import {HOSTURL, HOSTHEADER, KEYHEADER} from "../constants.js";
+import ResultList from "../Components/ResultList.jsx";
+import ErrorNotification from "../Components/ErrorNotification";
+
+class SearchContainer extends React.Component {
+    
+    constructor(props) {
+        super(props);
+        this.state= {
+            wordDetails: {},
+            error: {} 
+        }
+    }
+
+    getWordData = async (wordToSearch) => {
+        try {
+            const result =  await axios.get(`https://wordsapiv1.p.rapidapi.com/words/${wordToSearch}`, {
+            headers: {
+                'x-rapidapi-host': HOSTURL,
+                'x-rapidapi-key': process.env.REACT_APP_KEY
+            }})
+        this.setState({wordDetails: result.data.results})
+        this.setState({error:{}})
```

Comment:
you don't need to do this in one line`this.setState({wordDetails: result.data.results, error:{}})`
If you run setState twice, DOM would get re-rendered twice.